pkg/schema/schema.go

```diff
@@ -317,3 +328,90 @@ func validateResources(directory string) error {
 func CheckCatalogResources(manifestDir string) error {
 	return validateResources(manifestDir)
 }
+
+func validateExamplesAnnotations(csv *v1alpha1.ClusterServiceVersion) error {
+	var examples []CRDTemplate
+	var annotationsNames = []string{"alm-examples", "olm.examples"}
+	var found = false
+	annotations := csv.ObjectMeta.GetAnnotations()
+	// Return right away if no alm-examples are found.
+	if annotations == nil {
+		return fmt.Errorf("Unable to get annotations")
+	}
+
+	for _, name := range annotationsNames {
```

Comment:
I think you can clean this up a little bit:

```go
var annotationsExamples string
for _, name := range annotationsNames {
	annotationsExamples, ok := annotations[name]
	if !ok {
		continue
	}
}
if annotationsExamples == "" {
	return fmt.Errorf("Unable to get examples annotations")
}

// the rest
if json.Unmarshal([]byte(annotationsExamples), &examples) == nil {
//... 
}
```

it's a minor change but I think it makes it a little easier to read by removing the need for the `found` variable and letting you de-indent the rest of the function.